services/config/knack.py

```diff
@@ -176,6 +176,12 @@
             "layer_id": 0,
             "item_type": "layer",
         },
+        "view_3898": {
+            "description": "School Beacons EDP",
```

Comment:
can there be a better name for this that makes it clear why it's different from the "School Beacons" entry? 

why can't these two views be combined? 

why is this an internal only dataset?

Reply:
Looks like the dataset contains created by and modified by fields, that could be why?

Reply:
yes, its internal only because of PII. I dont know why they want those fields in there, I will ask Christina and Karo. There was an attempt to use the previous view, but for some reason they had to use a new one.